devtools/makefile_base_posix.mak

```diff
@@ -140,7 +140,7 @@ else ifeq ($(USE_VALVE_BINDIR),1)
 	CRYPTOPPDIR=linux32
 else
 	# Not using chroot, use old steam-runtime. (gcc 4.6.3)
-	export STEAM_RUNTIME_PATH ?= /valve/steam-runtime
+#	export STEAM_RUNTIME_PATH ?= /valve/steam-runtime
```

Comment:
This change should be left out, this is dead code anyway

Reply:
Building requires the NO_CHROOT flag afaik, but I can revert this and add this to build.sh envvars.

Reply:
Fixed